neuro_flow/cli/images.py

```diff
@@ -16,9 +16,16 @@
 
 
 @click.command()
+@option(
+    "-F",
```

Comment:
Usually this flag is called `-f/--force`. Do you have a reason for different naming?

Reply:
The only reason is to sync the flag naming with [neuro-extras](https://github.com/neuro-inc/neuro-extras/blob/master/neuro_extras/main.py#L104), but we could naturally call it `-f/--force` in neuro-flow.